packages/react/src/components/Card/__snapshots__/Card.story.storyshot

```diff
@@ -1457,6 +1457,7 @@ exports[`Storybook Snapshot tests and console checks Storyshots 1 - Watson IoT/C
                 </thead>
                 <tbody
                   aria-live="polite"
+                  className=""
```

Comment:
Not a big deal but it looks like we are applying a class attribute even if one is not provided.

Reply:
This is probably why we have so many snapshot changes

Reply:
@davidicus Thanks for the note! Yes, that was the reason why snapshots were updated. I added conditional props to avoid it.